### ISSUE TITLE
astrowidgets_viewer is not portable

### ISSUE THREAD

**Issue by pllim** [spacetelescope/dat_pyinthesky#16]
I cannot use it on Windows (or Linux for that matter):

```
[Voila] Voila listening on port ...
./astrowidgets_viewer: line 9: /Applications/Google Chrome.app/Contents/MacOS/Google Chrome: No such file or directory
```

This work is part of #15.

**Comment by pllim** [comment]
On Windows, I tried changing the offending line to this (in Git Bash shell):
```
/c/Program\ Files\ \(x86\)/Google/Chrome/Application/chrome.exe --app=http://localhost:8866
```
And get the following error:
```
VOILA: 16368
[Voila] Using C:\...\AppData\Local\Temp to store connection files
[Voila] Storing connection files in C:\...\AppData\Local\Temp\voila_rz22_17q.
[Voila] Serving static files from C:\...\Miniconda3\envs\py37\lib\site-packages\voila\static.
[Voila] Voila listening on port 8866.
[18780:12268:0404/134334.485:ERROR:cache_util_win.cc(19)] Unable to move the cache: 0
[18780:12268:0404/134334.486:ERROR:cache_util.cc(140)] Unable to move cache folder C:\...\AppData\Local\Google\Chrome\User Data\ShaderCache\GPUCache to C:\...\AppData\Local\Google\Chrome\User Data\ShaderCache\old_GPUCache_000
[18780:12268:0404/134334.486:ERROR:disk_cache.cc(184)] Unable to create cache
[18780:12268:0404/134334.486:ERROR:shader_disk_cache.cc(622)] Shader Cache Creation failed: -2
[Voila] WARNING | Notebook voilable_astrowidgets.ipynb is not trusted
Opening in existing browser session.
ERROR:tornado.application:Uncaught exception GET / (::1)
HTTPServerRequest(protocol='http', host='localhost:8866', method='GET', uri='/', version='HTTP/1.1', remote_ip='::1')
Traceback (most recent call last):
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\tornado\web.py", line 1592, in _execute
    result = yield result
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\tornado\gen.py", line 1133, in run
    value = future.result()
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\tornado\gen.py", line 1141, in run
    yielded = self.gen.throw(*exc_info)
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\voila\handler.py", line 64, in get
    kernel_id = yield tornado.gen.maybe_future(self.kernel_manager.start_kernel(kernel_name=kernel_name))
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\tornado\gen.py", line 1133, in run
    value = future.result()
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\tornado\gen.py", line 326, in wrapper
    yielded = next(result)
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\jupyter_server\services\kernels\kernelmanager.py", line 170, in start_kernel
    super(MappingKernelManager, self).start_kernel(**kwargs)
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\jupyter_client\multikernelmanager.py", line 110, in start_kernel
    km.start_kernel(**kwargs)
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\jupyter_client\manager.py", line 246, in start_kernel
    kernel_cmd = self.format_kernel_cmd(extra_arguments=extra_arguments)
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\jupyter_client\manager.py", line 170, in format_kernel_cmd
    cmd = self.kernel_spec.argv + extra_arguments
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\jupyter_client\manager.py", line 82, in kernel_spec
    self._kernel_spec = self.kernel_spec_manager.get_kernel_spec(self.kernel_name)
  File "C:\...\Miniconda3\envs\py37\lib\site-packages\jupyter_client\kernelspec.py", line 236, in get_kernel_spec
    raise NoSuchKernel(kernel_name)
jupyter_client.kernelspec.NoSuchKernel: No such kernel named astro36
ERROR:tornado.general:Could not open static file 'C:\\base\\images\\favicon.ico'
ERROR:tornado.general:Could not open static file 'C:\\style\\style.min.css'
ERROR:tornado.access:500 GET / (::1) 134.71ms
```

**Comment by pllim** [comment]
Fixed in #18.
